src/js/screens/CheckoutScreenOne.js

```diff
@@ -14,6 +14,7 @@ import SecondaryHeader from '../components/SecondaryHeader';
 import { ShoppingCart } from '../shopping-cart';
 import { SCREENS } from '../config/Constants';
 import { handleQuickActionsNavigation } from '../config/QuickActionsNavigation';
+import TestFairy from 'react-native-testfairy';
```

Comment:
Coding convension

Reply:
What do you mean? Should this line be different or put somewhere else?